controller/space.go

```diff
@@ -164,6 +184,39 @@ func (c *SpaceController) Delete(ctx *app.DeleteSpaceContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err.Error()))
 	}
+
+	spaceDeletionErrorExternal := fmt.Errorf("could not delete space")
+	spaceID, err := goauuid.FromString(ctx.SpaceID.String())
+	if err != nil {
+		log.Error(ctx.Context, map[string]interface{}{
+			"space_id": ctx.SpaceID,
+			"error":    err,
+		}, "could not convert the UUID of type github.com/satori/go.uuid to github.com/goadesign/goa/uuid")
+		return jsonapi.JSONErrorResponse(
+			ctx, goa.ErrInternal(spaceDeletionErrorExternal))
```

Comment:
```diff
-ctx, goa.ErrInternal(spaceDeletionErrorExternal))
+ctx, errors.NewInternalError(ctx, spaceDeletionErrorExternal))
```

rather changed it as above since `spaceDeletionErrorExternal` is already an error type